PCbuild/regen.targets

```
@@ -33,11 +33,15 @@
     <_JITOutputs Include="$(GeneratedPyConfigDir)jit_stencils.h"/>
     <_CasesSources Include="$(PySourcePath)Python\bytecodes.c;$(PySourcePath)Python\optimizer_bytecodes.c;"/>
     <_CasesOutputs Include="$(PySourcePath)Python\generated_cases.c.h;$(PySourcePath)Include\opcode_ids.h;$(PySourcePath)Include\internal\pycore_uop_ids.h;$(PySourcePath)Python\opcode_targets.h;$(PySourcePath)Include\internal\pycore_opcode_metadata.h;$(PySourcePath)Include\internal\pycore_uop_metadata.h;$(PySourcePath)Python\optimizer_cases.c.h;$(PySourcePath)Lib\_opcode_metadata.py"/>
+    <_SbomSources Include="$(PySourcePath)PCBuild\get_externals.bat" />
```

```suggestion
    <_SbomSources Include="$(PySourcePath)PCbuild\get_externals.bat" />
```

Incredibly minor, but we've had people run into case issues before, so may as well make it match.